config-linux.md

```diff
@@ -26,7 +26,7 @@ For more information, see the [namespaces(7)][namespaces.7_2] man page.
 Namespaces are specified as an array of entries inside the `namespaces` root field.
 The following parameters can be specified to set up namespaces:
 
-* **`type`** *(string, REQUIRED)* - namespace type. The following namespace types are supported:
+* **`type`** *(string, REQUIRED)* - namespace type. The following namespace types SHOULD be supported:
```

Comment:
In the runtime-spec PR, [you're claiming a config validation][1], but the "supported" language here sounds like it targets the runtime.  How about:

> Configuration SHOULD use, and runtimes SHOULD support, the following types:

Personally, I'd prefer MUSTing runtime support (more in #813), but even SHOULD would better than the current lack of RFC 2119 language.

[1]: https://github.com/opencontainers/runtime-tools/pull/486/files#diff-8351286f57eb81e245c9c99c07f3b34fR898